Hazelcast.Net/Hazelcast.Core/IMap.cs

```diff
@@ -294,6 +294,30 @@ public interface IMap<TKey, TValue> : IDistributedObject
         /// </remarks>
         void EvictAll();
 
+        /// <summary>Applies the user defined EntryProcessor to the entry mapped by the key.</summary>
+        /// <param name="key">key</param>
+        /// <param name="entryProcessor">the user defined the user defined <see cref="IEntryProcessor"/></param>
+        /// <returns>result of entry process</returns>
+        object ExecuteOnKey(TKey key, IEntryProcessor entryProcessor);
+
+        /// <summary>Applies the user defined EntryProcessor to the entries mapped by the collection of keys.</summary>
+        /// <param name="keys">keys to process</param>
+        /// <param name="entryProcessor">the user defined <see cref="IEntryProcessor"/></param>
+        /// <returns>the results mapped by each key in the collection.</returns>
+        IDictionary<TKey, object> ExecuteOnKeys(ISet<TKey> keys, IEntryProcessor entryProcessor);
+
+        /// <summary>Applies the user defined EntryProcessor to the all entries in the map.</summary>
+        /// <param name="entryProcessor">the user defined the user defined <see cref="IEntryProcessor"/></param>
```

Comment:
typo double "the user defined". I think these docs should also include the fact that the actual implementation must be on the server side.